app/screens/send-bitcoin-screen/use-fee.ts

```diff
@@ -13,20 +12,21 @@ import {
   useOnChainTxFeeLazyQuery,
 } from "@app/graphql/generated"
 import { gql } from "@apollo/client"
+import { GetFee } from "./payment-details/index.types"
 
-type FeeType = {
-  amount?: PaymentAmount<WalletCurrency>
-  status: "loading" | "error" | "unset" | "set"
-}
-
-type UseFeeInput = {
-  walletDescriptor: WalletDescriptor<WalletCurrency>
-  address: string
-  isNoAmountInvoice: boolean
-  invoice: string
-  paymentType: string
-  paymentAmount: PaymentAmount<WalletCurrency>
-}
+type FeeType =
+  | {
+      status: "loading" | "error" | "unset"
+      amount?: undefined | null
```

Comment:
no need for `amount`?

Reply:
If you want to be able to access `fee.amount` without going `if (amount in fee) fee.amount`, you have to set the property type as being undefined.